applications/GeoMechanicsApplication/custom_constitutive/coulomb_with_tension_cut_off_impl.cpp

```diff
@@ -158,4 +191,20 @@ void CoulombWithTensionCutOffImpl::load(Serializer& rSerializer)
     rSerializer.load("TensionCutOff", mTensionCutOff);
 }
 
+double CoulombWithTensionCutOffImpl::CalculateEquivalentPlasticStrain(const Vector& rSigmaTau,
+                                                                      CoulombYieldSurface::CoulombAveragingType AveragingType,
+                                                                      double lambda) const
+{
+    Vector dGdsigma = mCoulombYieldSurface.DerivativeOfFlowFunction(rSigmaTau, AveragingType);
+    double g1       = (dGdsigma[0] + dGdsigma[1]) * 0.5;
+    double g3       = (dGdsigma[0] - dGdsigma[1]) * 0.5;
+
+    double m_mean       = (g1 + g3) / 3.0;
+    double m_deviatoric = std::sqrt(std::pow(g1 - m_mean, 2) + std::pow(g3 - m_mean, 2));
```

Comment:
To adhere to the style guide, we don't need the extra m_ (but maybe it has another meaning that I missed )
```suggestion
    double mean       = (g1 + g3) / 3.0;
    double deviatoric = std::sqrt(std::pow(g1 - mean, 2) + std::pow(g3 - mean, 2));
```

Reply:
Here, "m" means the flow tensor in the principal basis. However, as it is confusing, and it has similar form with member variables, I removed "m_" from those variables.